### DISCUSSION TITLE
Anyone reproduce HPatches result?

### DISCUSSION THREAD

**Discussion by LinMenwill** [zju3dv/LoFTR#174] [Q&A]
I use the  "outdoor_ds.ckpt" loftr model to evaluate the haptches dataset, the images input are all resize to 480X640, but I can reproduce HPatches result in the paper, the results I reproduced are as follow:@3px:0.61470, @5px:0.69845, @10px:0.78132.

**Comment by HUSTNO1WXY** [comment]
I also can't reproduce Hpatches result, but code of this experiment is not released and I don't know how to reproduce it.
